solutions/kubernetes/functions.sh

```diff
@@ -18,12 +18,31 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-function add_separator {
-    set +xe
-    echo -e '\n\n\n========================= '"$*"' =========================\n'
+function add_primary_separator() {
+    printf "\n################################################################################\n"
+    printf "$*\n"
+    printf "################################################################################\n"
 }
 
-function validation {
+function add_secondary_separator() {
```

Comment:
Common separator is for different stages within secondary separator and check_status function to give clear overview of command completion: 
![image](https://user-images.githubusercontent.com/91177647/159908675-c58f88d2-91a8-43fd-8d8a-c7517495abbb.png)

Removing it might make it slightly harder to check for different commands execution at a glance.

Reply:
Ok. I just fill using three different separators is too much. However, we can keep it for now.

Reply:
By default add two steps to improve middle alignment which works for most, but for some statements which are small added tab spacing in the message itself when calling primary separator.